client/src/components/river/index.tsx

```diff
@@ -109,17 +113,26 @@ const FhirRiverView = (): React.ReactElement => {
 
   const handleClickCreateBatch = async (): Promise<void> => {
     setRunning(true);
+    const runOnce = selectedFrequency === "run once"
+    const route =  runOnce? "api/batch/" : "api/update-batch/";
```

Comment:
very nit: I would have put this ternary directly in the template string when calling axios